### ISSUE TITLE
Incorrect architecture for GameMode causes warnings in the logs

### ISSUE THREAD

**Issue by tannisroot** [lutris/lutris#1304]
Whenever I start any game with Feral Gamemode enabled (which is enabled for all games by default), I get this message spamming the terminal output (when Lutris is launched with `lutris -d`):
```
ERROR: ld.so: object '/usr/lib/libgamemodeauto.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS64): ignored.
```
This is quite annoying, and may cause some slow downs due to the spam. But disabling Gamemode gets rid of this.

**Comment by strycore** [comment]
I'm not sure this is fixable, the method documented on Feral's repo just doesn't work.

**Comment by Zeioth** [comment]
I was about to open pretty much the same issue:

I installed gamemode from the arch package from AUR.  I can run Lutris games with the checkbox "Enable Feral gamemode" checked.

But when I run any game, it doesn't seem to load it correctly (Lutris -d)
![gamemode](https://user-images.githubusercontent.com/3357792/48113600-ecf23e00-e25b-11e8-81c9-1b55da910dd0.png)

As you can see, I have /usr/lib/libgamemodeauto.so in the right location
![libs](https://user-images.githubusercontent.com/3357792/48113663-2e82e900-e25c-11e8-9e05-ada313689ad2.png)

**Comment by sigmaSd** [comment]
@Zeioth can you test with my PR https://github.com/lutris/lutris/pull/1305 I can't reproduce strycore issue

**Comment by sigmaSd** [comment]
I have another method to fix this but I really want to make sure that simple solution is not working

**Comment by strycore** [comment]
I still see the warnings on the current codebase

**Comment by tannisroot** [comment]
@strycore it's because you reversed that PR.

**Comment by strycore** [comment]
@tannisroot uh, nope, the current code has the `/usr/$LIB/libgamemodeauto.so.0` part, I have both 64 and 32bit builds of gamemode. I still have warnings on wine games.

**Comment by strycore** [comment]
Please send a PR if this still needs fixing.

**Comment by strycore** [comment]
Should be fixed by #2319 , please reopen if not the case.
